### PR TITLE
Feature/dependencies 2020 11 28

### PR DESCRIPTION
**Pull request by prescod**
Several dependency updates.

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -2,48 +2,48 @@
...
+cffi==1.14.3
 chardet==3.0.4
-click==7.1.1
-cryptography==2.9
+click==7.1.2
+cryptography==3.1.1
 defusedxml==0.6.0
-Django==2.2.11
-django-allauth==0.41.0
+Django==2.2.13
```
Comment (davisagli) [inline requirements/base.txt:18]: 2.2.16 is the latest version of Django 2.2

File: requirements/production.txt [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 # WSGI Handler
 # ------------------------------------------------
-gevent==1.4.0
+gevent==20.9.0
```
Comment (davisagli) [inline requirements/production.txt:9]: It doesn't appear that we're actually using gevent

File: requirements/test.txt [hunk 3]
```diff
@@ -3,14 +3,14 @@
 
 
 
-coverage==5.0.4
-flake8==3.7.9 # pyup: != 2.6.0
+coverage==5.3
+flake8==3.8.3 # pyup: != 2.6.0
```
Comment (davisagli) [inline requirements/test.txt:7]:
```suggestion
flake8==3.8.3
```

**Review by davisagli — CHANGES_REQUESTED** [review]

**Comment by davisagli** [comment]
Can we also try updating runtime.txt to python-3.8.6?

File: requirements/base.txt (hunk 1 above)
Comment (prescod) [reply requirements/base.txt]: https://github.com/SFDO-Tooling/sfdoc/pull/313/files#diff-7fe11226cff646f5d9f35faa76217059R18

File: requirements/production.txt (hunk 2 above)
Comment (prescod) [reply requirements/production.txt]: https://github.com/SFDO-Tooling/sfdoc/pull/313/files#diff-6f1f964e63cc3c4b164496bc75516b41L9

File: requirements/test.txt (hunk 3 above)
Comment (prescod) [reply requirements/test.txt]: https://github.com/SFDO-Tooling/sfdoc/pull/313/files#diff-6f1f964e63cc3c4b164496bc75516b41L9

**Comment by prescod** [comment]
https://github.com/SFDO-Tooling/sfdoc/pull/313/files#diff-fdc10ba93e16bd380f5462b0e821dc36

**Review by davisagli — APPROVED** [review]

**Pull request merged**